packages/react-app/src/components/AllowedMintCount.jsx

```diff
@@ -58,6 +58,20 @@ export default function AllowedMintCount() {
     }
   }, [allowedMinting, connectedAddress, localProvider])
 
+  useEffect(() => {
+    if (window.ethereum === undefined) return
+    const contractReference = externalContracts['5'].contracts.REMIX_REWARD
```

Comment:
you need to handle the case where the webapp is using the optimism network

Reply:
apologies. This is now done.